third_party/terraform/resources/resource_dataflow_job.go

```diff
@@ -58,113 +58,132 @@ func resourceDataflowJob() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 				// ForceNew applies to both stream and batch jobs
-				ForceNew: true,
+				ForceNew:    true,
+				Description: `A unique name for the resource, required by Dataflow.`,
 			},
 
 			"template_gcs_path": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The GCS path to the Dataflow job template.`,
 			},
 
 			"temp_gcs_location": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `A writeable location on GCS for the Dataflow job to dump its temporary data.`,
 			},
 
 			"zone": {
 				Type:     schema.TypeString,
 				Optional: true,
 				// ForceNew applies to both stream and batch jobs
-				ForceNew: true,
+				ForceNew:    true,
+				Description: `The zone in which the created job should run. If it is not provided, the provider zone is used.`,
 			},
 
 			"region": {
```

Comment:
Should this be added to the website documentation as well?

Reply:
`region` is not listed in https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs. Decided not add it in the website doc.